sklearn/utils/multiclass.py

```diff
@@ -311,7 +246,8 @@ def type_of_target(y):
     if y.dtype.kind == 'f' and np.any(y != y.astype(int)):
         return 'continuous' + suffix
     if len(np.unique(y)) <= 2:
-        assert not suffix, "2d binary array-like should be multilabel"
+        if suffix == "-multioutput":
+            return 'unknown'  # [[1, 2, 3]]
```

Comment:
my bad... :/

Also a related question what should the `type_of_target([[1, 2], [1, 2]])` be?

"binary-multioutput" (if there is anything like that?) or "multiclass-multioutput"  or simply binary?